app/views/vacancies/_filters.html.haml

```diff
@@ -11,7 +11,7 @@
 
     .govuk-form-group.govuk-inset-text#location-radius-select
       = label_tag 'radius', t('jobs.filters.within_radius'), class: 'govuk-label'
-      = select_tag 'radius', options_for_select(radius_filter_options, @vacancies_search.radius), disabled: @vacancies_search.disable_radius?, class: 'govuk-select govuk-!-width-full'
+      = select_tag 'radius', options_for_select(radius_filter_options, @vacancies_search.radius), disabled: @vacancies_search.disable_radius?, class: 'govuk-select govuk-!-width-full', 'data-radius': 10
```

Comment:
In rails you can do `{ data: { radius: 10 } }` for the same effect I _think_, but if this works it's fine with me

Reply:
your suggestion is neater